packages/application-config/test/fixtures/custom-applications/config-full.json

```diff
@@ -38,9 +38,8 @@
       "connect-src": ["https://track.avengers.app"]
     },
     "permissionsPolicies": {
-      "microphone": "()"
-    },
-    "strictTransportSecurity": ["includeSubDomains"]
+      "microphone": "*"
```

Comment:
Just to have it more explicit that it is a different value than the default one.